Openstack controller log support (#9115)
DataDog/integrations-core: openstack_controller/README.md

2. [Restart the Agent][3]

##### Log collection

1. Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml`:

   ```yaml
   logs_enabled: true
   ```

> 2. Add this configuration block to your `openstack_controller.d/conf.yaml` file to start collecting your Openstack Logs:

Reviewer: ```suggestion
2. Add this configuration block to your `openstack_controller.d/conf.yaml` file to start collecting your Openstack logs:
```

>    ```yaml
>    logs:
>      - type: file
>        path: "<LOG_FILE_PATH>"
>        source: openstack
>        service: "<SERVICE_NAME>"
>    ```

Reviewer: ```suggestion
   ```yaml
   logs:
     - type: file
       path: "<LOG_FILE_PATH>"
       source: openstack
   ```
```

>     Change the `path` and `service` parameter values and configure them for your environment. See the [sample openstack_controller.d/conf.yaml][7] for all available configuration options.

Reviewer: ```suggestion
    Change the `path` parameter value and configure them for your environment. See the [sample openstack_controller.d/conf.yaml][7] for all available configuration options.
```
